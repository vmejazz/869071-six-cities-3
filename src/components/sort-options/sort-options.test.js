import React from "react";
import renderer from "react-test-renderer";
import SortOptions from "../sort-options/sort-options.jsx";

it(`SortOption render`, () => {
  const tree = renderer
    .create(
        <SortOptions
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
