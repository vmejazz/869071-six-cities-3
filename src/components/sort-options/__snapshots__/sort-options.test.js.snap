// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortOption render 1`] = `
Array [
  <span
    className="places__sorting-type"
    tabIndex={0}
  >
    <svg
      className="places__sorting-arrow"
      height={4}
      width={7}
      xmlns="http://www.w3.org/2000/svg"
    >
      <use
        xlinkHref="#icon-arrow-select"
        xmlnsXlink="http://www.w3.org/1999/xlink"
      />
    </svg>
  </span>,
  <ul
    className="places__options places__options--custom places__options--opened"
  >
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex={0}
    >
      Popular
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex={0}
    >
      Price: low to high
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex={0}
    >
      Price: high to low
    </li>
    <li
      className="places__option"
      onClick={[Function]}
      tabIndex={0}
    >
      Top rated first
    </li>
  </ul>,
]
`;
